udata_schema_gouvfr/templates/modal.html

```diff
@@ -0,0 +1,63 @@
+{% for resource in dataset.resources %}
+  {% if resource.schema %}
+  <modal
+    v-ref:schema-modal-Id{{ resource.id | replace('-', '') }}
+    title="Schéma associé"
+    :visible="false">
+    <div class="modal-body">
+      <p>
+        Cette ressource est associée au schéma <b>{{ resource.schema }}</b> ce qui signifie qu'elle doit en respecter la spécification.
+      </p>
+
+      <h3>Documentation</h3>
+      <p>
+        Vous pouvez consulter la documentation de ce schéma, découvrir le contexte et les recommandations destinées aux producteurs de données.
+      </p>
+
+      <div>
+        <a
+          href="{{ documentation_urls[resource.id] }}"
+          rel="noopener noreferrer"
+          target="_blank"
+          title="Documentation du schéma {{ resource.schema }}"
+          class="btn btn-info"
+        >
+          <span class="fa fa-book"></span>
+          Lire la documentation
+        </a>
+      </div>
+
+
+      <h3>Validation</h3>
+      <div class="alert alert-info">
+        <p>
+          La validation d'une ressource par rapport à un schéma consiste à vérifier que la ressource est bien conforme au schéma qu'elle est censée respecter. En cas d'erreurs, un rapport de validation indique les erreurs trouvées : colonnes mal nommées, valeurs non conformes etc.
+        </p>
+      </div>
+
+      <p>
+        data.gouv.fr met à votre disposition l'outil <a href="https://validata.etalab.studio">Validata</a> pour valider vos fichiers par rapport à un schéma et corriger les éventuelles erreurs.
```

Comment:
```suggestion
        data.gouv.fr met à votre disposition un outil pour valider vos fichiers par rapport à un schéma et corriger les éventuelles erreurs.
```

Le double lien me parait porter à confusion.